man/systemd.network.xml

```diff
@@ -907,6 +907,31 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[Neighbor] Section Options</title>
+      <para>A <literal>[Neighbor]</literal> section accepts the
+      following keys. The neighbor section adds a permanent, static
+      entry to the neighbor table (IPv6) or arp table (IPv4) for
```

Comment:
acronyms are usally uppercased in our docs, i.e. `arp` → `ARP`, please

Reply:
Done